bigquery/bigquery_storage_quickstart/main.go

```diff
@@ -223,15 +226,35 @@ func processStream(ctx context.Context, client *bqStorage.BigQueryReadClient, st
 			r, err := rowStream.Recv()
 			if err == io.EOF {
 				return nil
-			}
-			if err != nil {
-				retries++
-				if retries >= retryLimit {
-					return fmt.Errorf("processStream retries exhausted: %v", err)
+			} else if err != nil {
+				// If there is an error, check whether it is a retryable error with a retry delay and sleep instead of increasing retries count.
```

Comment:
Please wrap this comment to under 100 characters.

Reply:
Done